README.md

```diff
@@ -94,6 +94,14 @@ attack command:
     	Attack name
   -output string
     	Output file (default "stdout")
+  -prom-bind
```

Comment:
What do you think of replacing these four flags with a single `-prometheus-url`? Example `-prometheus-url=http://0.0.0.0:8880/metrics`. It being defined would imply it's enabled. By default, it'd be empty.

Reply:
done